js/cliffChart.js

```diff
@@ -0,0 +1,319 @@
+/*******************************************
+ *               CliffChart.js             *
+ *******************************************/
+
+/**
+ * A simple stick figure path with a full circle head,
+ * from y=-30 (head) to y=0 (feet).
+ */
+const STICK_FIGURE_PATH = `
+  M 0 -30
+  m -5 0
+  a 5 5 0 1 1 10 0
+  a 5 5 0 1 1 -10 0
+
+  M 0 -25
+  L 0 -5
+
+  M 0 -20
+  L -5 -15
+  M 0 -20
+  L 5 -15
+
+  M 0 -5
+  L -5 0
+  M 0 -5
+  L 5 0
+`;
+
+class CliffChart {
+    /**
+     * @param {Array} data - e.g. [{ year: 1900, deathCount: 10 }, ... ]
+     *   Each object needs { year: Number, deathCount: Number }.
+     */
+    constructor(membersData) {
+        // 1) Store data and sort by year
+        this.data = this.prepareData(membersData);;
```

Comment:
Currently, membersData is being modified directly which is impacting other visuals like the death rate chart. Try making a copy of membersData first before passing into this.prepareData. 

You can do the following:
`this.data = this.prepareData(membersData.map(d =>({...d})));`